client/my-sites/site-settings/style.scss

```diff
@@ -440,6 +440,49 @@
 	& + & {
 		margin-top: 24px;
 	}
+
+	&.disconnect {
+		width: 70%;
+		margin: auto;
+		height: 100px;
+
+		@include breakpoint( "<480px" ) {
+			padding: 12px;
+			max-width: 200px;
+		}
+
+		@include breakpoint( "<660px" ) {
+			margin-top: 1em;
+			max-width: 400px;
```

Comment:
Just wondering - will this override the previous `max-width` in `<480px` since it appears later, or does that breakpoint block has higher specificity?